patches/@types__css-tree@2.3.1.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/index.d.ts b/index.d.ts
+index e6a09d2756a7d942d7dbf2d6b9437deb06e8eb31..41cf6b6efdea7e800c77866c1e103c7be4b5c68f 100755
+--- a/index.d.ts
```

Comment:
`@types/css-tree` is little better now. But at the same time missing some things that we need. So, did a patch instead of a copy.